sceptre/cli.py

```diff
@@ -630,6 +630,11 @@ def init_project(ctx, project_name):
         folder_path = os.path.join(project_folder, folder)
         os.makedirs(folder_path)
 
+    # Create .gitignore file in project
+    gitignore_path = os.path.join(project_folder, ".gitignore")
+    with open(gitignore_path, "wt") as gitignore_file:
+        gitignore_file.write("*.pyc")
+
```

Comment:
This should probably be a flag instead (--gitignore and -g) of the default behavior (not all templates are version controlled, not everyone uses git, etc).

Reply:
I'm not sure I agree with this.

The spirit of the feature is to avoid forgetting to add a `.gitignore` file, if the flag has to be added then it's just as easy to forget and the feature might be overlooked by most people. Or in other terms, it's almost as easy to add a `.gitignore` file as it is to remember to add the flag.